tests/model_fields/tests.py

```diff
@@ -236,7 +236,7 @@ def test_abstract_model_pending_lookups(self):
         )
 
 
-class DateTimeFieldTests(unittest.TestCase):
+class DateTimeFieldTests(test.TestCase):
```

Comment:
could you fix the other test cases in this file and make it a separate commit? I see at least `BooleanFieldTests` that has database activity but uses `unittest.TestCase`.